addons/attach/CfgVehicles.hpp

```diff
@@ -86,6 +86,10 @@ class CfgVehicles {
             dayLight = 0;
             onlyInNvg = 1;
             useFlare = 0;
+            maxLifetime = 8 * 60 * 60;
+            blinkingPattern[] = {0.1, 0.9}; // 0.1 s on, 0.9 s off
```

Comment:
```suggestion
            blinkingPattern[] = {0.1, 1.1}; // 0.1 s on, 1.1 s off
```

According to the [MS-2000 manual](http://www.ostrich.co.jp/wps/wp-content/uploads/2016/11/manual-y1-03-0275b.pdf) the blink rate should be 50±10 flashes per minute. I've checked out your branch and I have to say I prefer `blinkingPattern[] = {0.1, 1.1};`, which would meet that spec, seems a little less "nervous".